src/kernels/composable_kernel/include/utility/amd_buffer_addressing.hpp

```diff
@@ -358,15 +284,11 @@ __device__ ushort amd_buffer_load<ushort, 1>(const ushort* p_src_block,
     // fill in byte 3
     src_block_config.range[3] = 0x00027000;
 
-#if !CK_WORKAROUND_SWDEV_231101
     index_t src_thread_addr_offset = src_thread_data_offset * sizeof(ushort);
     index_t src_const_addr_offset  = src_const_data_offset * sizeof(ushort);
 
-    return __llvm_amdgcn_buffer_load_bf16(
-        src_block_config.data, 0, src_thread_addr_offset + src_const_addr_offset, false, false);
-#else
-    return p_src_block[src_thread_data_offset + src_const_data_offset];
-#endif
+    return __llvm_amdgcn_raw_buffer_load_bf16(
```

Comment:
@asroy same here. it's better to comment that we are hard-coding Soffset as 0 at this moment.

Reply:
fixed